saleor/graphql/payment/types.py

```diff
@@ -434,26 +426,11 @@ def resolve_events(root: models.TransactionItem, info):
         return TransactionEventByTransactionIdLoader(info.context).load(root.id)
 
     @staticmethod
-    def resolve_user(root: models.TransactionItem, info):
-        def _resolve_user(event_user):
-            requester = get_user_or_app_from_context(info.context)
-            if (
-                requester == event_user
-                or requester.has_perm(AccountPermissions.MANAGE_USERS)
-                or requester.has_perm(AccountPermissions.MANAGE_STAFF)
-            ):
-                return event_user
-            return None
-
-        if not root.user_id:
-            return None
-
-        return UserByUserIdLoader(info.context).load(root.user_id).then(_resolve_user)
-
-    @staticmethod
-    def resolve_app(root: models.TransactionItem, info):
+    def resolve_created_by(root: models.TransactionItem, info):
```

Comment:
Shouldn't be wrapped in any permissions?

Reply:
https://github.com/saleor/saleor/pull/11534#discussion_r1054132965